src/components/Calculator.js

```diff
@@ -1,44 +1,72 @@
 import React from 'react';
+import calculate from '../logic/calculate';
 
 class Calculator extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      total: null,
+      next: null,
+      operation: null,
+    };
+    this.handleClick = this.handleClick.bind(this);
+    this.displayValues = this.displayValues.bind(this);
+  }
+
+  handleClick(btnName) {
+    const newState = calculate(this.state, btnName);
+    this.setState(() => ({
+      total: newState.total,
+      next: newState.next,
+      operation: newState.operation,
+    }));
+  }
```

Comment:
- [ ] Your application crashes when I press operation buttons (+, -, *, ÷) consecutively, e.g if I want to do an addition and press `+`, and later change my mind to do a subtraction, if I pressed `-`, or any other operation button, your application crashes with the error below (see image). Kindly fix it.
<img width="1055" alt="Screen Shot 2021-10-22 at 3 37 20 AM" src="https://user-images.githubusercontent.com/47994600/138389566-bf2d4e3f-778e-4e88-9228-84a5a1f67f8b.png">